docs/source/extension/extension_migration.rst

```diff
@@ -4,7 +4,22 @@
 .. _extension_migration:
 
 Extension Migration Guide
-================================================
+=========================
+
+JupyterLab 4.2 to 4.3
+---------------------
+
+CSS styling
+^^^^^^^^^^^
+
+Previously JupyterLab was leaking CSS rules globally. Starting from 4.3, this is not the case
+any more. The side effects for extensions are:
+
+- for DOM elements attached outside the application shell may have broken styles. To fix this,
+  you should add the class ``jp-ThemedContainer`` to the DOM elements added outside the application shell.
+- DOM elements ``code``, ``kbd``, ``pre``, ``samp`` and ``tt`` may have broken styles. To fix this,
+  prepend the class ``.jp-ThemedContainer`` to your rule; e.g.
+  ``.jp-Inspector-content pre`` becomes ``.jp-ThemedContainer .jp-Inspector-content pre``
```

Comment:
Is it still accurate after adding `.jp-ThemedContainer` to the `body`? Or does it only pertain custom shells?

Reply:
It is unfortunately for all cases because it changes the CSS selectivity of the rules that therefore may shadow previously applied rule. See for example in this PR, commits 943516a735bc4fc60cce34bcc47f7d9213a98df2 and 1b66679ceffc28493af232ae7ac493c1e0eac5a6